MonstercatNet/Models/Playlist/Playlist.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SoftThorn.MonstercatNet.Models.Playlist
+{
+    public sealed class Playlist
+    {
+        public string Name { get; set; }
+        public bool Public { get; set; }
+        public PlaylistTrack[] Tracks { get; set; }
```

Comment:
[nullable reference types](https://docs.microsoft.com/en-us/dotnet/csharp/nullable-references) are enabled for this repo, so this property generates a warning. Can we set an initial value here, so that it is not null?